### ISSUE TITLE
erro in      return self.log_odds(y) + pred

### ISSUE THREAD

**Issue by 123carmen** [Ekeany/XGBoost-From-Scratch#4]
In file Naive-Gradient-Boosting.py,--line-210
please tell me what is this 'y' in    
     return self.log_odds(y) + pred

full code is here 
    def predict(self, X):
        
        pred = np.zeros(X.shape[0])
        for estimator in self.estimators:
            pred += self.learning_rate * estimator.predict(X)
            
        return self.log_odds(y) + pred

thank you for your code,
